spark/v3.3/spark/src/test/java/org/apache/iceberg/TestFileIOSerialization.java

```diff
@@ -101,6 +104,60 @@ public void testHadoopFileIOJavaSerialization() throws IOException, ClassNotFoun
     Assert.assertEquals("Conf values must be present", "v2", actualConf.get("k2"));
   }
 
+  @Test
+  public void testResolvingFileIOKryoSerialization() throws IOException {
```

Comment:
Why are these tests in Spark instead of in the module that defines each FileIO?

Reply:
This was  a pre-exisitng test class `TestFileIOSerialization` checking the ser/de for HadoopFileIO, hence added other fileIO's serialization tests also to the same class. 

> the module that defines each FileIO

I thing what we can do here is create the kryoHelper (which doesn't take dependency on spark) in core and since  integrations (i.e aws etc) take a dependency on it, it would be available to all FileIO.

The only concern I had here was that these test can become outdated when spark uses diff version of twitter-chill or uses diff lib like [margo](https://github.com/magro/kryo-serializers).

Reply:
Kryo should work out of the box without chill (see my comment above), so it shouldn't be an issue.

Can you add the Kryo test helper to `org.apache.iceberg.TestHelpers`?